run/awake_start

```diff
@@ -27,7 +27,42 @@ function ping {
       curl -s -H "Authorization: Bearer $TESLAFI_API_TOKEN" "https://www.teslafi.com/feed.php?command=wake_up"
       # No need to contact TeslaFi too often to prevent API lockout (max 3 calls/min).
       sleep 600
-    fi    
+    fi
+
+    if [[ -n "${TESSIE_API_TOKEN:+x}" ]]
+    # Use Tessie API to keep car awake
+    then
+        http_response=$(curl -s -w "%{http_code}" \
+            -H "Authorization: Bearer $TESSIE_API_TOKEN" \
+            -H "Accept: application/json" \
+            -H "User-Agent: github.com/marcone/teslausb" \
+            "https://api.tessie.com/$TESSIE_VIN/wake")
+        
+        # Extract HTTP status code
+        http_status="${http_response: -3}"
+        
+        # Extract response body (excluding status code)
+        response_body="${http_response:0:${#http_response}-3}"
+        
+        # Check if HTTP status code is not 200 (OK)
+        if [ "$http_status" != "200" ]; then
```

Comment:
"then" should go on the next line to match existing code style (also in two more places below)